runner/prepare_benchmark.py

```diff
@@ -291,6 +292,7 @@ def beeline(query):
       "/root/convert_to_parquet.py")
     ssh_spark("/root/spark/bin/spark-submit --master spark://%s:7077 /root/convert_to_parquet.py" % opts.spark_host)
 
+    ssh_spark("/root/spark/sbin/stop-thriftserver.sh")
```

Comment:
I updated the commit to fix this.

In Github pull requests, is it preferred to update the original commit, like I did, or to add an additional commit that contains the fix?